pkg/tekton/syntax/pipeline.go

```diff
@@ -1114,23 +1116,54 @@ func (j *ParsedPipeline) GenerateCRDs(pipelineIdentifier string, buildIdentifier
 		if err != nil {
 			return nil, nil, nil, err
 		}
+
 		previousStage = stage
 
+		pipelineTasks := createPipelineTasks(stage, p.Spec.Resources[0].Name)
+
 		linearTasks := stage.getLinearTasks()
-		for _, lt := range linearTasks {
+
+		for index, lt := range linearTasks {
+			wipeOutput := wipeOutput(stage, lt.Name, index, len(linearTasks), isLastStage)
+			if wipeOutput {
```

Comment:
Could just be `if wipeOutput(...) {`

Reply:
I'd rather go for the variable, is more clear. But I'll change it.